CHANGEDB.php

```diff
@@ -1012,4 +1012,5 @@
 UPDATE `gibbonAlertType` SET `automatic`='Y' WHERE type='Core';end
 UPDATE `gibbonNotificationEvent` SET event='New Global Alert' WHERE event='New Student Alert' AND moduleName='Student Alerts';end
 INSERT INTO `gibbonNotificationEvent` (`event`, `moduleName`, `actionName`, `type`, `scopes`, `active`) VALUES ('New Class Alert', 'Student Alerts', 'Manage Student Alerts', 'Core', 'All,gibbonPersonIDStudent,gibbonYearGroupID', 'Y');end
-";
+INSERT INTO `gibbonLanguage` (`gibbonLanguageID`, `name`) VALUES (NULL, 'Balochi');end
```

Comment:
Wonderful, thanks Ali! I've made one small tweak, to change this to an INSERT IGNORE statement. This will help for the occasions where someone has already manually added the language to their system, the next time they update it will prevent an error based on unique keys.
```suggestion
INSERT IGNORE INTO `gibbonLanguage` (`gibbonLanguageID`, `name`) VALUES (NULL, 'Balochi');end
```